### PR TITLE
Installing an Ecosystem using Helm

### PR DESCRIPTION
**Pull request by CaroMac**
Updates to installing and architecture documentation

### CODE REVIEW

File: src/markdown-pages/docs/ecosystem/ecosystem-installing-k8s.md [hunk 1]
```diff
@@ -101,25 +264,46 @@ To upgrade the Galasa Ecosystem to use a newer version of Galasa, for example ve
...
+
+Follow the installation instructions in the [Configuring Ingress](#configuring-ingress) section to update the rest of your `values.yaml` file, including values to configure Ingress and Dex.
+
+After updating your `values.yaml` file, run the following command, providing the path to the <a href=https://github.com/galasa-dev/helm/tree/main/charts/ecosystem target="_blank"> ecosystem</a> directory in the helm repository, for example, `~/helm/charts/ecosystem`.
+
+```console
+helm install <release-name> /path/to/helm/charts/ecosystem --wait
+```
+
+When the `helm install` command completes with a successful deployment message, follow the installation instructions that are provided in the [Verifying the installation](#verifying-your-galasa-ecosystem-installation) section to test the deployed Ecosystem using `helm test` and determine the bootstrap URL.
```
Comment (eamansour) [inline src/markdown-pages/docs/ecosystem/ecosystem-installing-k8s.md:302]: Might be worth removing this or making it clear that installing the development version isn't something that people would typically use. In most cases, you would only use the development version if you are working on the helm chart.

File: src/markdown-pages/docs/ecosystem/ecosystem-overview.svg
Comment (eamansour) [inline src/markdown-pages/docs/ecosystem/ecosystem-overview.svg:1]: Would it be worth indicating the microservices belong to the Galasa Ecosystem within the kubernetes namespace, like in the original image? Also, I think moving the "Kubernetes cluster" and "Kubernetes namespace" labels to the top left of each of the boxes would make it look a bit nicer.

File: src/markdown-pages/docs/ecosystem/galasa-ecosystem-architecture.svg
Comment (eamansour) [inline src/markdown-pages/docs/ecosystem/galasa-ecosystem-architecture.svg:1]: Might be worth making the boxes more equal in width, moving the "Test engines" away from the other services, and making the "Interacts with" label a bit less cramped in the new image.

File: src/markdown-pages/docs/ecosystem/ecosystem-installing-k8s.md (hunk 1 above)
Comment (CaroMac) [reply src/markdown-pages/docs/ecosystem/ecosystem-installing-k8s.md:302]: will remove the Development section.

File: src/markdown-pages/docs/ecosystem/ecosystem-overview.svg
Comment (CaroMac) [reply src/markdown-pages/docs/ecosystem/ecosystem-overview.svg:1]: added dotted box around as per discussion

File: src/markdown-pages/docs/ecosystem/galasa-ecosystem-architecture.svg
Comment (CaroMac) [reply src/markdown-pages/docs/ecosystem/galasa-ecosystem-architecture.svg:1]: have remodelled as per discussion

**Review by eamansour — APPROVED** [review]

**Pull request merged**